### PR TITLE
Update Pin Configuration with respect to SRA Board 2.4

### PR DESCRIPTION
**Pull request by gautam-dev-maker**
* Update the LSA pin configuration 
* Remove APIs for Motor B
* Change conditions in Bar Graph

### CODE REVIEW

File: src/adc.c [hunk 1]
```diff
@@ -78,20 +79,24 @@ int read_adc(int adc_pin)
 {
     if (adc_pin == LSA_A0)
     {
-        return adc1_get_raw(ADC_CHANNEL_6);
+        return adc1_get_raw(ADC_CHANNEL_4);
```
Comment (VedantParanjape) [inline src/adc.c:82]: Verify this again, running on the kit.

**Review by VedantParanjape — CHANGES_REQUESTED** [review]

File: src/adc.c (hunk 1 above)
Comment (gautam-dev-maker) [reply src/adc.c:82]: Yes, We have again verified the ADC channels and pin configuration

File: src/bar_graph.c [hunk 2]
```diff
@@ -31,70 +31,31 @@ static int enabled_bar_graph_flag = 0;
...
-    // motor driver a is off and motor driver b is in normal mode, so we can use IN1 - IN4 pins
-    else if (read_motor_driver_mode(a) == 0 && read_motor_driver_mode(b) == 2)
-    {
-        bit_mask = (1ULL << BG_LED_1) | (1ULL << BG_LED_2) | (1ULL << BG_LED_3) | (1ULL << BG_LED_4);
-        enabled_bar_graph_flag = 3;
+        enabled_bar_graph_flag = 0;
     }
-    // motor driver a is in parallel mode and motor driver b is off, so we can use IN1, IN2, IN5 - IN8 pins
-    else if (read_motor_driver_mode(a) == 1 && read_motor_driver_mode(b) == 0)
+    // motor driver a is in parallel mode, so we can use IN1, IN2, IN5 - IN8 pins
```
[the remark this answers is not in the document]
Comment (gautam-dev-maker) [reply src/bar_graph.c:48]: So far, we have tested the following cases:
1) bar graph and motor driver(NORMAL Mode)
2)bar graph and motor driver (PARALLEL Mode)

File: include/lsa.h [hunk 3]
```diff
@@ -39,6 +39,7 @@ typedef union line_sensor_array
         int A1;
         int A2;
         int A3;
+        int A4;
     } lsa;
 
     int adc_reading[4];
```
[the remark this answers is not in the document]
Comment (gautam-dev-maker) [reply include/lsa.h]: Corrected this bug

File: include/bar_graph.h [hunk 4]
```diff
@@ -62,22 +56,21 @@ esp_err_t enable_bar_graph();
...
  * Motor Driver A: Normal
- * Motor Driver B: Off
- * Mode = 7, so for mode = 7,
- * mask = bitmask[7] = 0x0F = 00001111
+ * Mode = 2, so for mode = 2,
+ * mask = bitmask[2] = 0x0F = 00001111
  * data = 0xAA = 10101010
  * 
- * Since, motor driver A is in normal mode and motor driver B is off, we can only use IN5 - IN8 pins of the bar graph LED
+ * Since, motor driver A is in normal mode, we can only use IN1, IN5, IN6, IN8 pins of the bar graph LED
```
[the remark this answers is not in the document]
Comment (gautam-dev-maker) [reply include/bar_graph.h]: Corrected the comment

**Pull request merged**